crates/ruma-events-macros/src/event_content.rs

```diff
@@ -204,6 +204,19 @@ pub fn expand_event_content(
             quote! { true }
         };
 
+        let initializer = if kept_redacted_fields.is_empty() {
+            quote! {
+                impl #redacted_ident {
+                    #[doc = #doc]
```

Comment:
This is the docstring for the type, the constructor should have its own (like `Creates an empty {type_name}`).